app/main/createWindow.js

```diff
@@ -41,6 +52,9 @@ export default ({ src, isDev }) => {
   // Setup event listeners for main window
   globalShortcut.register(shortcut, toggleMainWindow)
 
+  // Set the theme vibrancy from config
+  mainWindow.setVibrancy(themeMode(config.get('theme')))
```

Comment:
This is not supported on Windows etc. right? Will this just be ignored there or what?